src/pages/index.js

```diff
@@ -29,51 +43,124 @@ const renderCard = (item) => {
 };
 
 //create objects
+//api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7ff811c0-9f09-42c2-975e-cca434c12992",
+    "content-type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+  },
+});
+
 //profile section
-const userInformation = new UserInfo(".profile");
+const userInfo = new UserInfo(".profile");
+
+const profileSection = new Section(
```

Comment:
This code doesn't work

Please, delete it

`Section` is only for rendering cards